pytorch_lightning/trainer/properties.py

```diff
@@ -97,6 +97,10 @@ def world_size(self) -> int:
         # some training types define a world size
         return getattr(self.accelerator.training_type_plugin, "world_size", 1)
 
+    @property
+    def should_rank_save_checkpoint(self) -> bool:
+        return self.accelerator.training_type_plugin.should_rank_save_checkpoint
+
```

Comment:
Are you sure it's okay adding a convenience function in the trainer for it?

Do you expect/allow users to use/rely on it directly?

Reply:
Yeah, I don't expect it to be used by users, but for internal use.

Reply:
My point is that if it is for internal use, the property should be prepended by `_`.

It's ugly accessing an underscore attribute but we need a mechanism to avoid having to deprecate later.